LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -236,17 +273,25 @@ class LockboxXCUITests: BaseTestCase {
         XCTAssertEqual(app.switches["sendUsageData.switch"].value as? String, "1")
     }
 
-    func test7EntryDetails() {
+    func test6SortEntries() {
         navigator.goto(Screen.LockboxMainPage)
-        waitforExistence(app.tables.cells.staticTexts["iosmztest@gmail.com"])
-        app.tables.cells.staticTexts["iosmztest@gmail.com"].tap()
-        XCTAssertTrue(app.tables.cells.staticTexts["Web Address"].exists)
-        XCTAssertTrue(app.tables.cells.staticTexts["Username"].exists)
-        XCTAssertTrue(app.tables.cells.staticTexts["iosmztest@gmail.com"].exists)
-        XCTAssertTrue(app.tables.cells.staticTexts["Password"].exists)
+        waitforExistence(app.navigationBars["Firefox Lockbox"], timeout: 10)
```

Comment:
Do we want an `accessibilityIdentifier` for this navigation bar, also?

Reply:
I have already modified this, is the same here, that is already an `accessibilityIdentifier`, but with the name not following the pattern.